commands/moderation/kick.js

```diff
@@ -0,0 +1,109 @@
+const Discord = require('discord.js');
+var dateFormat = require('dateformat');
+
+module.exports = {
+    name: "kick",
```

Comment:
I didn't get to test this one out because I'd already booted the two alt users. It looks like it will work fine though.